loadbalancer/algorithm.go

```diff
@@ -186,7 +220,7 @@ func (ch *consistentHash) Apply(ctx *routing.LBContext) routing.LBEndpoint {
 	key := net.RemoteHost(ctx.Request).String()
 	if _, err := h.Write([]byte(key)); err != nil {
 		log.Errorf("Failed to write '%s' into hash: %v", key, err)
-		return ctx.Route.LBEndpoints[rand.Intn(len(ctx.Route.LBEndpoints))]
+		return ctx.Route.LBEndpoints[ch.sr.getIntn(len(ctx.Route.LBEndpoints))]
```

Comment:
`Write` actually never returns error https://golang.org/src/hash/fnv/fnv.go?#L109 so this path is a kind of fallback that is never executed, Therefore there is no need to have `safeRandom` in every `consistentHash` since is never used.